test/integration/fixtures/blocks/core__table__deprecated-1.html

```diff
@@ -1,3 +1,3 @@
-<!-- wp:table {"backgroundColor":"subtle-pale-blue"} -->
+<!-- wp:table {"hasFixedLayout":false,"backgroundColor":"subtle-pale-blue"} -->
```

Comment:
This change means the fixture no longer represents as accurately what the block attributes were at the time it was the current version. 

In this case, the deprecated version relied on the default for `hasFixedLayout` being `false.`. As each deprecation should bring the block up to the latest version including changing the default for `hasFixedLayout`, leaving this as it was should ensure the deprecation is working correctly.